OpenRA.Mods.Common/Scripting/Global/UtilsGlobal.cs

```diff
@@ -54,6 +55,20 @@ public bool All(LuaValue[] collection, LuaFunction func)
 			return true;
 		}
 
+		[Desc("Returns the original collection filtered with the func.")]
+		public LuaTable Where(LuaValue[] collection, LuaFunction func)
+		{
+			var t = Context.CreateTable();
+
+			foreach (var c in collection)
+				using (var ret = func.Call(c))
```

Comment:
Um, that doesn't look like the `Where` function, but rather like `Any`...
Anyway, I could do that here in theory, but should I? I mean we don't do this anywhere here.